builder/requirements.txt

```diff
@@ -0,0 +1,14 @@
+pyarrow
+pandas
+anndata
+numpy
+tiledb
+tiledbsoma
```

Comment:
Latest pip release is not currently not compatible with latest code in this repo. As documented for notebooks usage, I had to build TileDB-SOMA from source and locally install. We might want to document similar steps for builders, even if it's just a temporary state of affairs, if only for the SCB team's benefit.

Reply:
we will have a pip-installable tiledbsoma in a few days.  So I'd rather not change this.

Could you clarify this:
> Latest pip release is not currently not compatible with latest code in this repo

What is the pip version, and what is the error?

Reply:
```
❯ python -m builder ~/cxg/cell-census/data/cell-census-small/ -v build --manifest ~/dev/cxg/cell-census/data/manifest-small.csv
Traceback (most recent call last):
  File "/Users/atolopko/miniforge3/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Users/atolopko/miniforge3/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/Users/atolopko/dev/cxg/cell-census/cell-census/builder/__main__.py", line 12, in <module>
    from .anndata import open_anndata
  File "/Users/atolopko/dev/cxg/cell-census/cell-census/builder/anndata.py", line 8, in <module>
    from .datasets import Dataset
  File "/Users/atolopko/dev/cxg/cell-census/cell-census/builder/datasets.py", line 66, in <module>
    def create_dataset_manifest(info_collection: soma.Collection, datasets: List[Dataset]) -> None:
AttributeError: module 'tiledbsoma' has no attribute 'Collection'. Did you mean: 'SOMACollection'?

❯ pip list | grep soma
cell-census            0.0.1.dev0  /Users/atolopko/dev/cxg/cell-census/soma-scratch/experiments/cell_census/src
tiledbsoma             0.1.18
```

Reply:
ah, this is due to a missing version specifier - you are picking up a very old version of tiledbsoma which _is_ on PyPi.  Good catch.